lib/collections/ordered.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (collection) => {
   return collection.getAll().sort((a, b) => {
-    if (a.data.order && b.data.order) { // Sort by order value, if given
+    if (a.data.order !== undefined && b.data.order !== undefined) { // Sort by order value, if given
```

Comment:
```suggestion
    if (typeof a.data.order !== 'undefined' && typeof b.data.order !== 'undefined') { // Sort by order value, if given
```

I _think_ this is still the preferred way of checking for undefined.